recipe/meta.yaml

```diff
@@ -28,12 +31,17 @@ test:
     - test
   imports:
     - uuid6
+  requires:
```

Comment:
Oh, I know what'd going on. You have two `requires`.

Reply:
Oops! Totally missed that. Fixed now - should be good :)